addons/web/static/src/js/fields/relational_fields.js

```diff
@@ -1036,8 +1036,9 @@ const Many2OneAvatar = FieldMany2One.extend({
      * @override
      */
     _render() {
+        const fieldName = ['res.partner', 'res.users', 'hr.employee'].includes(this.field.relation) ? 'avatar_128' : 'image_128';
```

Comment:
Always use `avatar_128` here. This is a widget called `Many2OneAvatar`, models that don't have the avatar field should not use it.
If we someone need to support another field at some point, it should be a parameter of the widget, not hardcoded like this.